retriever/lib/fetch.py

```diff
@@ -0,0 +1,30 @@
+
+import os
+import sqlite3
+import pandas as pd
+from retriever.engines import sqlite
+from retriever.lib.defaults import HOME_DIR
+from retriever.lib.install import install_sqlite
+from retriever.lib.download import download
+
+
+
+def fetch(dataset):
+    """Install dataset into pandas dataframes.
+       each file in a separate dataframe
+    """
+    #download(dataset)
+    installation_path = os.path.join(HOME_DIR,'{}.sqlite'.format(dataset))
+    install_sqlite(dataset, file=installation_path)
```

Comment:
Since we're just creating this and then deleting it what about an in memory database instead. It's quicker and doesn't require the clean up?

Reply:
Hi @ethanwhite  thanks for your review. I fixed all the issues pointed by your comments.
The only thing remaining is creating in memory db. Is there an already implemented method as the current method `install_sqlite` doesn't support in memory db